packages/@yodaos/speech-synthesis/src/pcm-player.cc

```diff
@@ -89,10 +86,12 @@ void PcmPlayer::end() {
     }
     int err;
     fprintf(stdout, "[PcmPlayer] draining player\n");
-    if (pa_simple_drain(stream, &err) < 0) {
-      fprintf(stderr, "[PcmPlayer] drain player error(%d): %s\n", err,
-              pa_strerror(err));
-    }
+    do {
+      if (pa_simple_drain(stream, &err) < 0) {
+        fprintf(stderr, "[PcmPlayer] drain player error(%d): %s\n", err,
+                pa_strerror(err));
+      }
+    } while (err == /** drain timed out, retrying */ 8);
```

Comment:
Do we have a enum instead?

Reply:
I'm afraid no.

https://freedesktop.org/software/pulseaudio/doxygen/simple_8h_source.html